src/ResourceManager/KeyVault/Commands.KeyVault/Commands/RemoveAzureKeyVaultCertificate.cs

```diff
@@ -0,0 +1,90 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Globalization;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.KeyVault.Models;
+using KeyVaultProperties = Microsoft.Azure.Commands.KeyVault.Properties;
+using Microsoft.Azure.KeyVault.Models;
+
+namespace Microsoft.Azure.Commands.KeyVault
+{
+    /// <summary>
+    /// The Remove-AzureKeyVaultCertificate cmdlet deletes a certificate in an Azure Key Vault. 
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, CmdletNoun.AzureKeyVaultCertificate,
+        SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.High,
+        HelpUri = Constants.KeyVaultHelpUri)]
+    public class RemoveAzureKeyVaultCertificate : KeyVaultCmdletBase
+    {
+        #region Input Parameter Definitions
+
+        /// <summary>
+        /// VaultName
+        /// </summary>
+        [Parameter(Mandatory = true,
+                   Position = 0,
+                   ValueFromPipelineByPropertyName = true,
+                   HelpMessage = "Specifies the name of the vault to which this cmdlet adds the certificate.")]
+        [ValidateNotNullOrEmpty]
+        [ValidatePattern(Constants.VaultNameRegExString)]
+        public string VaultName { get; set; }
+
+        /// <summary>
+        /// Name
+        /// </summary>       
+        [Parameter(Mandatory = true,
+                   Position = 1,
+                   ValueFromPipelineByPropertyName = true,
+                   HelpMessage = "Specifies the name of the certificate in key vault.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// If present, do not ask for confirmation
+        /// </summary>
+        [Parameter(HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
```

Comment:
Is a Force parameter really necessary?  If there are no special conditions for deletion that you need to prompt the user for, there is no real need for a Force parameter